charts/aac-manage-case-assignment/values.preview.template.yaml

```diff
@@ -12,4 +12,4 @@ java:
         - notify-mca-api-key-secret
   environment:
     S2S_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
-    CCD_DATA_STORE_API_BASE_URL: http://ccd-data-store-api-pr-601.service.core-compute-preview.internal
+    CCD_DATA_STORE_API_BASE_URL: http://ccd-data-store-api-pr-1187.service.core-compute-preview.internal
```

Comment:
Are these changes related or should they be in a separate review ?

Reply:
They'll be reverted before merging 👍 Just temporarily pointing to the PRs where I've made similar changes to ensure it all works e2e.